metadata/report/etcd/report.go

```diff
@@ -52,14 +53,28 @@ type etcdMetadataReport struct {
 
 // GetAppMetadata get metadata info from etcd
 func (e *etcdMetadataReport) GetAppMetadata(metadataIdentifier *identifier.SubscriberMetadataIdentifier) (*common.MetadataInfo, error) {
-	// TODO will implement
-	panic("implement me")
+	key := e.getNodeKey(metadataIdentifier)
+	_data, err := e.client.Get(key)
+	if err != nil {
+		return nil, err
+	}
+	info := &common.MetadataInfo{}
+	err = json.Unmarshal([]byte(_data), info)
+	if err != nil {
```

Comment:
Why does it start with ’_‘ ？Is it golang's style?.. ：）

Reply:
It should be "data". Starting with '_' is just a self-style when naming []byte type variables